components/generic/SmartLink.vue

```diff
@@ -45,6 +48,14 @@
           });
         }
         return this.destination;
+      },
+      isExternalLink() {
+        if (!process.browser) return;
+
+        const destDomain = parseDomain(this.destination);
+        const currentDomain = parseDomain(window.location.href);
+
+        return destDomain && currentDomain && destDomain.domain + destDomain.tld !== currentDomain.domain + currentDomain.tld;
```

Comment:
This is hard to read, is there another way to write it? Maybe using brackets would help?